uber/models.py

```diff
@@ -700,13 +700,15 @@ def delete_from_group(self, attendee, group):
             self.delete(attendee)
             group.attendees.remove(attendee)
 
-        def assign_badges(self, group, new_badge_count, new_badge_type=None, **extra_create_args):
-            new_badge_type = new_badge_type or c.ATTENDEE_BADGE
+        def assign_badges(self, group, new_badge_count, new_badge_type=c.ATTENDEE_BADGE, new_ribbon_type=c.NO_RIBBON, **extra_create_args):
             diff = int(new_badge_count) - group.badges
             sorted_unassigned = sorted(group.floating, key=lambda a: a.registered, reverse=True)
+
+            ribbon_to_use = group.new_ribbon if group.new_ribbon is not c.NO_RIBBON else new_ribbon_type
```

Comment:
I realize that this is kind of nit-picky, since at the moment ``group.new_ribbon`` only checks dealers anyway, but I intended ``group.new_ribbon`` to be overridable in plugins, since e.g. MAGFest probably wants to add logic for stuff like band ribbons and such, so I think this will be relevant.